src/main/java/com/openshift/client/utils/TarFileUtils.java

```diff
@@ -35,7 +35,7 @@ public static boolean hasGitFolder(InputStream inputStream) throws IOException {
 		try {
 			boolean gitFolderPresent = false;
 			tarInputStream = new TarArchiveInputStream(new GzipCompressorInputStream(inputStream));
-			for (TarArchiveEntry entry = null; (entry = tarInputStream.getNextTarEntry()) != null;) {
```

Comment:
this is a bad change: removig initialization, in what way should this be better?